src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentPresentation/IPresentationParams.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.LanguageServer.Protocol;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.DocumentPresentation
+{
+    internal interface IPresentationParams
```

Comment:
Oh interesting, are you not able to use VS' protocol types for these?

Reply:
Not sure what you mean. This interface is just to allow the base handler to work on Text and Uri presentation requests, because they share some properties. Everything that implements this is still VS protocol types.

Reply:
Ahhhhh ok